templates/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
```suggestion
<html dir="ltr" lang="en">
```

Reply:
Oh interesting. Good note, thank you. 👍

Reply:
Np, I just learned about it the other day. Def gonna include it from here on out